integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/common/AvailabilityPollingUtils.java

```diff
@@ -48,4 +77,92 @@
             }
         };
     }
+
+    /**
+     * Check whether server is being started
+     *
+     * @param host Host name to check the availability
+     * @param port Port number to check the availability
+     * @return Whether the server is started or not
+     */
+    public static Callable<Boolean> isHostAvailable(final String host, final int port) {
+
+        return new Callable<Boolean>() {
+            @Override
+            public Boolean call() {
+
+                log.info("Checking whether the vfs server is started.");
```

Comment:
since this is related to check common server start shall we use common log entry